dbfit-java/informix/src/main/java/dbfit/environment/BigDecimalExactClassTransformer.java

```diff
@@ -0,0 +1,21 @@
+package dbfit.environment;
+
+import java.sql.SQLException;
+
+import dbfit.util.TypeTransformer;
+
+public class BigDecimalExactClassTransformer implements TypeTransformer {
+
+    @Override
+    public Object transform(Object o) throws SQLException {
```

Comment:
BTW  - no statements in the implementation are throwing `SQLException` so I think we can remove that `throws` clause here (and the related import).